### PR TITLE
Grove pi motors

### PR DESCRIPTION
**Pull request by aninternetof**
Part of #163.

This moves the motor control from the Adafruit GPIO library to the GrovePi library.

### CODE REVIEW

**Comment by aninternetof** [comment]
Not sure what is up with coveralls and this repo. The coverage is still at 100%.

File: .travis.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ before_install:
   libreadline-dev
 - sudo apt-get install -qq libsqlite3-dev libxml2 libxml2-dev libssl-dev libbz2-dev
   wget curl llvm
-- cd Adafruit_Python_GPIO/
+- cd GrovePi/Software/Python
```
Comment (cabarnes) [inline .travis.yml]: Can this be a `pushd/popd`?

File: README.md [hunk 2]
```diff
@@ -10,12 +10,7 @@
...
 
-Rovercode runs on any single-board-computer supported by the [Adafruit Python GPIO wrapper library](https://github.com/adafruit/Adafruit_Python_GPIO), including the NextThingCo CHIP, Raspberry Pi, and BeagleBone Black. Once installed, head to to rovercode.com and connect to your rover.
-
-To learn about the other pieces of rovercode, visit our [architecture documentation](https://contributor-docs.rovercode.com/architecture.html), or start at [the root of rovercode's documentation](https://contributor-docs.rovercode.com).k
-
-## Contributing
-Check out our [contributing page](http://rovercode.readthedocs.io/en/latest/contribute.html) to get started.
+Rovercode runs on a Raspberry Pi 3 with the [GrovePi+ sensor board](https://www.seeedstudio.com/GrovePi-p-2241.html) and the [Grove I2C motor controller board](https://www.seeedstudio.com/Grove-I2C-Motor-Driver-p-907.html).
```
Comment (cabarnes) [inline README.md]: A link to the Raspberry Pi 3 would be good as well

**Comment by cabarnes** [comment]
Maybe coveralls is having an issue because it's not called right after the tests. There's some documentation building and it changes directories a couple times before it is called.

File: README.md (hunk 2 above)
Comment (aninternetof) [reply README.md]: Good idea. Done.

File: .travis.yml (hunk 1 above)
Comment (aninternetof) [reply .travis.yml]: Good catch! Fixed.

**Comment by aninternetof** [comment]
Other than the Coveralls strangeness, any other comments on this PR?

**Comment by cabarnes** [comment]
Nope, looks good to me

**Pull request merged**